testing/testing.go

```diff
@@ -110,3 +114,20 @@ func EnsureNodeSync(nodeManager common.NodeManager) {
 		}
 	}
 }
+
+// GetNetworkID returns appropriate network id for test based on
+// default or provided -network flag.
+func GetNetworkID() int {
+	switch strings.ToLower(*networkSelected) {
+	case fmt.Sprintf("%d", params.MainNetworkID), "mainnet":
+		return params.MainNetworkID
+	case fmt.Sprintf("%d", params.RinkebyNetworkID), "rinkeby":
+		return params.RinkebyNetworkID
+	case fmt.Sprintf("%d", params.RopstenNetworkID), "ropsten":
```

Comment:
Let's switch to `testnet` in compliance with what `geth` accepts. Sometimes it's just more consistent (I faced it while deploying our cluster and passing arguments).

Reply:
You mean return `testnet` as default?

Reply:
Ah, bad wording again. I meant to rename.

Reply:
Ah! You mean renaming `ropsten` to `testnet`?

Reply:
Yep.

Reply:
Ok. Because of others who may be new, i will just have `testnet` has a option for `ropsten`, so others if they use `roptsten` can still get this working without issue.

Reply:
Yep, sounds good.